### PR TITLE
docs(keycloak): improve quick-setup

### PR DESCRIPTION
**Pull request by gcusnieux**
https://github.com/gravitee-io/_support_team/issues/193

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.z6.web.core.windows.net/support-193-keycloak/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-bqcmfiaulu.chromatic.com)

### CODE REVIEW

File: docker/quick-setup/keycloak/README.md [hunk 1]
```diff
@@ -12,14 +12,33 @@ Keycloak UI is accessible here http://localhost:8081/auth
...
-In the Console UI (https://docs.gravitee.io/apim/3.x/apim_installguide_authentication_keycloak.html#apim_console_configuration):
-- Go to Organization/Settings
-- Configure the Identity provider following documentation [http://localhost:8081/auth/realms/master/.well-known/openid-configuration](Configuration URLs can be find here)
-
-You should now be able to use it to log to Console UI.
+- Read how to create a new client for your application. (Already imported with `realm/realm-gio.json`)
+- Create a user in the "GIO" realm.
+
+You can read how to [https://docs.gravitee.io/apim/3.x/apim_installguide_authentication_keycloak.html#apim_console_configuration](configure the Identity provider following documentation) 
+but it's already configure with docker environment.
```
Comment (ytvnr) [inline docker/quick-setup/keycloak/README.md]:
```suggestion
but it's already configured with docker environment.
```

Comment (ytvnr) [inline docker/quick-setup/keycloak/README.md]: should be written in one line no ?

**Review by ytvnr — APPROVED** [review]
awesome !
looks good to me

**Review by Okhelifi — APPROVED** [review]
👏🏼

**Pull request merged**